### DISCUSSION TITLE
What mean is this 'Your verification is pending'?

### DISCUSSION THREAD

**Discussion by kimhaming** [community/community#121781] [GitHub Education]
### Hi everyone,

The submission was successful, and when I access the application page, I see the phrase "Your confirmation is pending" under the profile, is it going well? Should I wait?

**Comment by gurjeetsinghvirdee** [comment]
Hey @kimhaming 

After submitting your application for the GitHub Student Developer Pack, it's normal to see `Your confirmation is pending` on the application page. This means GitHub is verifying your student status. You'll receive an email when your benefits are ready to claim. Just wait for the approval email and follow the instructions to start enjoying your benefits
